local-volume/provisioner/README.md

```diff
@@ -0,0 +1,53 @@
+# local-volume-provisioner
+
+NOTE: provisioner is still in development and is not functional yet
+
+local-volume-provisioner is an out-of-tree static provisioner for the Local volume plugin, which is a 1.7 alpha feature.
+
+It runs on each node in the cluster and monitors specified directories to look for new local mount points.  It then statically creates a Local PersistentVolume for each mount point.  It also monitors when the PersistentVolumes have been deleted, and will clean up the volume.
```

Comment:
I thought a little more about this situation.  I'm not sure if we want to automatically delete the PV object if the cleanup fails.  Because if the cleanup failed for some reason not due to path missing, then if we deleted the PV object, we would then automatically recycle a partially cleaned up PV that can be used by some other application.

So I think in the case of where cleanup fails, we should throw an error on the PV, and require admin manual intervention to cleanup.